### DISCUSSION TITLE
Convert Task to Issue in different repo

### DISCUSSION THREAD

**Discussion by alexkgold** [community/community#4723] [Projects and Issues]
We use a single repo as our "epics" repo and have the tasks in their respective repos. I would love to be able to convert a task list in my epic issue into an issue in a different repo.

**Comment by daanleenders** [comment]
Would love to be able to do this more easily. But in my experience you are able to do this with a bit of a work around. You create the issue initially in the current repo and then move it to it respective repo. The link will stay intact.

**Reply by alexkgold** [reply]
That's a great point. Would still love the ability to do it directly, but this is a good workaround for the time-being.

**Comment by Mats-Jakobsson** [comment]
Hi,

When you say move, @daanleenders, do you mean transfer issue (or is there a way of moving issues that I miss.)? If I do so, the link between the parent and child seems to be lost. The initial tasklist item in the parent issues looses its description and becomes a #<number> entry without any link.

Thanks!
